Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -177,9 +231,14 @@ public class MongoDatabase {
     }
 
     /// Initializes a new `MongoDatabase` instance, not meant to be instantiated directly.
-    internal init(fromDatabase: OpaquePointer, withClient: MongoClient) {
+    internal init(fromDatabase: OpaquePointer,
```

Comment:
nit with existing code: I think the slightly more swifty way of naming these would be something like 
```swift
init(fromDatabase database: OpaquePointer,
     withClient client: MongoClient,
     ...
```

Reply:
fixed